toolkit/tools/internal/packagerepo/repoutils/repoutils.go

```diff
@@ -157,3 +158,24 @@ func verifyClonedRepoContents(clonedRepoContents, expectedPackages []*repocloner
 
 	return
 }
+
+// FindCreateRepoCommand searches for createrepo or createrepo_c in $PATH and returns the first one found
+func FindCreateRepoCommand() (cmd string, err error) {
+	creatrepo_cmds := []string{"createrepo", "createrepo_c"}
```

Comment:
I would request you to please reorder as, the c version of create repo is faster than the python implementation and we should prefer the `c` version if avalible.

```suggestion
	creatrepo_cmds := []string{"createrepo_c", "createrepo"}
```